src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,17 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, some street, #12-34, 541289";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses has to be in this format (BLOCK, STREET, UNIT, POSTAL_CODE)";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+\\,.+\\,.+\\,.+";
 
     public final String value;
     private boolean isPrivate;
+    
+    private int block;
+    private String street;
+    private String unit;
```

Comment:
oh!! took me a while to realize, you need to create multiple classes that make up the address class; this solution is incorrect.
please read the activity description carefully.